tests/app/test_party_service.py

```diff
@@ -0,0 +1,46 @@
+import unittest
+from unittest import mock
+from app.services.party_service import PartyService
+import requests
+
+
+class PartyBusinessTestHelper:
+
+    def __init__(self, status_code, reason, text):
+        self.status_code = status_code
+        self.reason= reason
+        self.text = text
+
+
+class PartyTestCase(unittest.TestCase):
+    """Test cases for party service"""
+
+    @staticmethod
+    def test_get_business_details_sends_request_to_remote_party_service():
+        """Test get business details sends a request"""
+        sut = PartyService()
+        business_data = PartyBusinessTestHelper(200, "OK", '{"something": "else"}')
+        requests.get = mock.Mock(name='get', return_value=business_data)
```

Comment:
If we use the requests_mock package here we wouldn't need to define the helper function and could do it all over one line.

The current method would make it tricky to carry out tests which hit two separate requests objects as well